presto-docs/src/main/sphinx/release/release-0.247.rst

```diff
@@ -4,34 +4,37 @@ Release 0.247
 
 **Highlights**
 ==============
+* Add the interface `QueryInterceptor` to allow for custom logic to be executed before or after query execution. (:pr:`15565`)
+* Add support for temporary (session-scoped) functions
```

Comment:
`.` at the end of every line (also in all the sections below). see https://prestodb.io/docs/current/release/release-0.246.html